benefits/core/migrations/0002_data.py

```diff
@@ -177,7 +177,7 @@ def load_data(app, *args, **kwargs):
         auth_provider=None,
         selection_label_template="eligibility/includes/selection-label--mst-courtesy-card.html",
         start_template="eligibility/start--mst-courtesy-card.html",
-        form_title=_("Confirm your Courtesy Card"),
+        form_title=_("Courtesy Card information"),
```

Comment:
@indexing So this is the title for the page with the Courtesy Card form. The title should now be `Agency card information | Cal-ITP Benefits` correct?

<img width="1187" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/731c8a42-6f6c-4d46-95b3-59c0b59c43c5">

Reply:
Updating to Agency card information.